### PR TITLE
Support to private access feature

### PR DESCRIPTION
**Pull request by leonel1524**
[#146](https://github.com/adempiere/adempiere-vue/issues/146) Hello everyone, in this PR, the private access feature on the records is added

### CODE REVIEW

**Comment by yamelsenih** [comment]
Hi @leonel1524 please add a gif for it

**Comment by leonel1524** [comment]
> Hi @leonel1524 please add a gif for it

adempiere-vue-private-access

**Comment by yamelsenih** [comment]
Hi @leonel1524 please can you add standard notification for user with "Record Locked" and "Record Unloked"?

Best Regard

**Comment by leonel1524** [comment]
> Hi @leonel1524 please can you add standard notification for user with "Record Locked" and "Record Unloked"?
> 
> Best Regard

Hello @yamelsenih, this request has been added
adempiere-vue-lock-record-notification

File: src/components/ADempiere/ContextMenu/contextMenuMixin.js [hunk 1]
```diff
@@ -450,6 +466,19 @@ export const contextMixin = {
         type: 'success',
         duration: 1500
       })
+    },
+    validatePrivateAccess(response) {
+      if (response.isLocked) {
+        this.actions.find(item => item.action === 'unlockRecord').hidden = false
```
Comment (yamelsenih) [inline src/components/ADempiere/ContextMenu/contextMenuMixin.js:505]:
```Javascript
let value = this.actions.find(item => item.action === 'unlockRecord')
value.hidden = false
value.tableName = response.tableName
...
```

Comment (yamelsenih) [reply src/components/ADempiere/ContextMenu/contextMenuMixin.js:505]: Test with it

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/ContextMenu/contextMenuMixin.js (hunk 1 above)
Comment (leonel1524) [reply src/components/ADempiere/ContextMenu/contextMenuMixin.js:505]: solved

**Pull request merged**